DigitalLearningSolutions.Data.Migrations/202307181334_updateTermsAndConditions.cs

```diff
@@ -0,0 +1,26 @@
 personal data. This will vary from organisation to organisation depending on setting and context. It is each organisation’s responsibility to ensure they comply with all relevant National, International and local legislation and guidance.</li></ul></li><li><strong>System Support</strong><ul data-list-level=""1""><li>ALL learner support is provided through our network of Centres&nbsp;(visit the&nbsp;<a title=""admin"" href=""https://www.dls.nhs.uk/findyourcentre"">Find Your Centre</a>&nbsp;page for contact details).</li><li>System support / Super Administration is provided by Health Education England. The service is provided 5 working days a week 9 am to 5 pm excluding English bank holidays. A reduced service is provided over each Christmas holiday period and this is notified to all organisations in advance. This team does not provide direct support to learners.</li><li>The support service can be contacted via the support ticket process found within the system itself. If the Tracking System is unavailable,&nbsp;Centre Administrators can email&nbsp;<a href=""mailto:dls@hee.nhs.uk"">dls@hee.nhs.uk</a>.</li><li>Health Education England undertake to respond to your request within two working days, with one of the following responses:<ul data-list-level=""2""><li>The issue is now resolved</li><li>More information from the organisation is required</li><li>The issue requires software development</li><li>A request for change needs to be submitted and considered</li></ul></li><li>The Super Administrator will have full, unrestricted access to your Digital Learning Solutions products to enable support to be carried out in a speedy and efficient manner. The Super Administrator will remind you that they are accessing the products in this manner.</li></ul></li><li><strong>System Governance</strong><ul data-list-level=""1""><li>The Digital Learning Solutions Expert Advisory Group (DLSEAG) provide the governance for the system. Governance will include:<ul data-list-level=""2""><li>Review and critique materials proposing new functionality</li><li>Evaluate and prioritise requests for system changes</li><li>Provide expert knowledge, guidance and skills to determine the future development of the system</li></ul></li></ul></li></ol><ol start=""11"" type=""1""><li><strong>Warranties</strong><ul data-list-level=""1""><li>Health Education England does not give any warranty as to the accuracy of the information recorded in the system by organisations or individuals. It is the express responsibility of organisations to verify, by their own means, the accuracy of the information entered.</li></ul></li><li><strong>System Decommissioning</strong><ul data-list-level=""1""><li>If the system is decommissioned the data held will be returned to the originating organisation. If a specific format is required this must be discussed with the System Support to ensure that if it is feasible. The data will be transferred securely and with send and received receipts.</li></ul></li></ol>'
```

Comment:
don't set the updated date in the down script.

Reply:
Done